README.md

```diff
@@ -137,7 +137,7 @@ to pass if you want your changes to be merged.
 | Issue Type        | Action           |
 | ---|---|
 | Package is out of date      | Create a Pull Request or Issue |
-| New desired package     | Create a Pull Request or Issue |
+| New desired package     | Follow [Package Inclusion Guide](./docs/package-inclusion-guide.md) |

```

Comment:
I feel we should have separate rows for "Add a package to my Cloud Shell" which links to the "Add a package into Cloud Shell" guide.  And then another row with "Add a package to everyone's Cloud Shell" which links to the "Package Inclusion Guide".  Thoughts?

Reply:
Sure! We can do that!